test_data/session_test.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
Suggest changing the session file name should be changed to something specific describing whats special about it rather than something generic.